modules/claims_api/app/controllers/concerns/claims_api/endpoint_deprecation.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module EndpointDeprecation
+    extend ActiveSupport::Concern
+
+    included do
+      V0_DEV_DOCS = 'https://developer.va.gov/explore/benefits/docs/claims?version=0.0.1'
+      V1_DEV_DOCS = 'https://developer.va.gov/explore/benefits/docs/claims?version=current'
```

Comment:
i wonder if its worth using
https://developer.va.gov/explore/benefits/docs/claims?version=1.0.0
as the url instead.

the above url still redirects to "current" it looks like, but should stay v1 if a v2 is introduced as the new "current" version.

Reply:
Ah, ok.
Yeah, I had originally put `version=1.0.0` but when it redirected to "current", I wasn't sure if maybe that url would "break" whenever a new version comes out.
Updated [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6289/commits/8aa37123798926ef1d594dbe486052c0d97392d5).